lib/mayaUsd/ufe/trf/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# -----------------------------------------------------------------------------
+# sources
+# -----------------------------------------------------------------------------
+target_sources(${PROJECT_NAME} 
+    PRIVATE
+        RotationUtils.cpp
+        UsdTransform3dFallbackMayaXformStack.cpp
+        UsdTransform3dMayaXformStack.cpp
+        XformOpUtils.cpp
```

Comment:
In MayaUsd these are the transform3d files left behind that are Maya specific. Moved them into "trf" sub-folder to match the ones moved to UsdUfe (in ufe/trf folder).